tools/Google.Cloud.Tools.Analyzers/MissingCopyrightNoticeAnalyzer.cs

```diff
@@ -93,7 +93,8 @@ public class MissingCopyrightNoticeAnalyzer : DiagnosticAnalyzer
                 "//------------------------------------------------------------------------------\n// <auto-generated>",
                 "//------------------------------------------------------------------------------\r\n// <auto-generated>",
                 "// Generated by the protocol buffer compiler.  DO NOT EDIT!",
-                "// Generated by the MSBuild WriteCodeFragment class.");
+                "// Generated by the MSBuild WriteCodeFragment class.",
+                "// This file was automatically created from");
```

Comment:
These are matched with StartsWith checks. I guess the field name is misleading. I'll rename it to `AutoGenerationCommentPrefixes`. I didn't want to complete the comment in this case because I think going forward it is reasonable to assume anything that starts with this text means the file was auto-generated.